meteor-app/imports/api/parts/parts.js

```diff
@@ -0,0 +1,44 @@
+import { Mongo } from 'meteor/mongo'
```

Comment:
Please name this file `imports/api/parts/schema.js` instead of `imports/api/parts/parts.js`

If you don't, then you have a duplication when you import it, which looks weird